package.json

```diff
@@ -8,6 +8,10 @@
     "bootstrap": "^4.3.1",
```

Comment:
may be dont use bootstrap for starter

Reply:
I agree, it was called but not used, I removed it.